Sources/DistributedSystem/ChannelCompression.swift

```diff
@@ -28,8 +28,7 @@ final class ChannelCompressionHandshakeServer: ChannelInboundHandler, RemovableC
 
     func channelActive(context: ChannelHandlerContext) {
         timer = context.eventLoop.scheduleTask(in: DistributedSystem.pingInterval*2) {
-            let channel = context.channel
-            self.logger.info("Session timeout for client @ \(channel.remoteAddressDescription), close connection.")
+            self.logger.info("\(context.channel.addressDescription): session timeout for compression client handshake, close connection")
```

Comment:
Maybe add a metadata entry with this address so it will be automatically added to every log message?